css/style.css

```
@@ -98,3 +101,59 @@ li {
   width: 2rem;
   height: 2rem;
 }
+
+.content {
+  font-size: 9rem;
+  line-height: 1.4;
+  text-align: center;
+  color: #fff;
+  padding-left: 32rem;
+  padding-right: 32rem;
+  text-shadow: 1px 1px 2px black;
+}
+
+.promo-decoration {
+  padding-top: 3.2rem;
+}
+
+IMG.decoration {
```

No need to link it to `img`
```suggestion
decoration {
```